src/app/containers/Disclaimer/index.test.jsx

```diff
@@ -50,7 +54,7 @@ describe('Disclaimer Component', () => {
     const { getByText } = renderComponent();
     expect(
       getByText(
-        'Приложение Русской службы BBC News доступно для и . Грузите его на ваш девайс и продолжайте получать новости от Би-би-си.',
+        'Приложение Русской службы BBC News доступно для IOS и Android. Вы можете также подписаться на наш канал в Telegram.',
```

Comment:
This test is going to fail. For some reason, `getByText` does not pick up the entire text :/ . I believe this getByText should test for: `Приложение Русской службы BBC News доступно для и . Вы можете также подписаться на наш канал в .`
Otherwise, great job 👍